class UsuarioNoEncontradoError(Exception):
    """Clase base para excepciones en el m√≥dulo. La utilizmos en caso de que no se encuentre el usuario en usuarios_log"""
    pass